vhdbuilder/packer/install-dependencies.sh

```diff
@@ -321,6 +322,7 @@ if [ $OS == $UBUNTU_OS_NAME ] && [ $(isARM64)  != 1 ] && [ $UBUNTU_MAJOR_VERSION
   installAndConfigureArtifactStreaming acr-mirror-${UBUNTU_RELEASE//.} deb
 fi
 
+# Need to enable Artifact Streaming for AzureLinux 3.0
```

Comment:
can we add an explicit TODO here to make sure we come back to this?

Reply:
Added a TODO - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/c7910902c25648bf307f6e335ae3a1b541856398)